wandb/sdk/wandb_artifacts.py

```diff
@@ -1544,11 +1584,12 @@ def init_gcs(self) -> "gcs_module.client.Client":
         self._client = storage.Client()
         return self._client
 
-    def _parse_uri(self, uri: str) -> Tuple[str, str]:
+    def _parse_uri(self, uri: str) -> Tuple[str, str, Optional[str]]:
         url = urlparse(uri)
         bucket = url.netloc
         key = url.path[1:]
-        return bucket, key
+        version = url.fragment if url.fragment else None
```

Comment:
what is the falsey value of `url.fragment`? i.e. do you need the if/else?

Reply:
The falsey value is `''`, but I need it to be `None` hence this ugly ternary op.